### DISCUSSION TITLE
Display total project time on the frontend of a website?

### DISCUSSION THREAD

**Discussion by dev-elop** [kimai/kimai#4217] [General]
Hi there,

I am wondering if it is possible to display the total time worked on a specific project on the frontend of a website?

I am wanting to display the total time and to have it automatically update when timings are saved (it doesn't necessarily have to be a live frontend timer, but would be pretty awesome if it did that).

Is this at all possible?

Thanks!

**Comment by kevinpapst** [comment]
> on the frontend of a website

Whatever that means, but the API does not have endpoints for statistics.
